tests/epyccel/test_epyccel_dicts.py

```diff
@@ -70,21 +70,31 @@ def kwarg_init():
     assert isinstance(python_result, type(pyccel_result))
     assert python_result == pyccel_result
 
-def test_pop_element(python_only_language):
+@pytest.mark.parametrize("language", (
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("python", marks = pytest.mark.python)
+    )
+)
+def test_pop_element(language):
```

Comment:
There is a shortcut for this if you want:
```suggestion
def test_pop_element(stc_language):
```

This can help us find methods that we still have to translate to Fortran